README.md

```diff
@@ -102,6 +102,22 @@ Best practices for working with JOSE and Decentralized Identifiers.
 }
 ```
 
+#### JWS
+```json
+{
+  "payload": "AXESIK2wLYjbI6_7RSmECitaiWNg3KizdXHKTezdEDMHpAqT",
+  "signatures": [
+    {
+      "signature": "ay5XcKLR01VMt6_agZFHxwCATEvGkzB_lBz-qfKfQwZKtn13BeL-qrHXExyzn_zEI9MmmX4RhcAdZLrbFbbjBw",
+      "protected": {
```

Comment:
This is not valid JWS. `protected` must be base64url encoded. Was this produced by some library?

Reply:
Yeah you're right. Was a while ago I created this PR, maybe I did this to just make it clear what's inside. Should definitely update it to be base64url encoded.
No library that I'm aware of produces unencoded protected headers like this.